tests/view_tests/tests/test_debug.py

```diff
@@ -1272,3 +1275,13 @@ def test_cleanse_setting_recurses_in_dictionary(self):
         initial = {'login': 'cooper', 'password': 'secret'}
         expected = {'login': 'cooper', 'password': CLEANSED_SUBSTITUTE}
         self.assertEqual(cleanse_setting('SETTING_NAME', initial), expected)
+
+
+class DecoratorsTestCase(SimpleTestCase):
+    def test_sensitive_variables_incorrect_usage(self):
+        with self.assertRaises(TypeError):
+            sensitive_variables(lambda: None)
+
+    def test_sensitive_post_parameters_incorrect_usage(self):
+        with self.assertRaises(TypeError):
+            sensitive_post_parameters(lambda: None)
```

Comment:
I think it's better to test a "normal" function instead of `lambda`, e.g.
```python
with self.assertRaisesMessage(TypeError, msg).
    @sensitive_variables
    def test_func(self, password):
        pass
```